packages/alert/src/Alert.tsx

```diff
@@ -50,6 +58,26 @@ export interface AlertProps extends React.HTMLAttributes<HTMLElement> {
    * The variant to use. Defaults to "relative".
    */
   type?: 'relative' | 'toast' | 'banner'
+
+  /**
+   * Show / Hide close button
+   */
+  showCloseButton?: boolean
+
+  /**
+   * Show / Hide Alert
+   */
+  isOpen?: boolean
+
+  /**
+   * Show alert duration
+   */
+  delay?: number
```

Comment:
Can we change the types to ensure `onClose` is present if we specify a type `toast`? If not, this could cause a runtime error that TS could easily catch.

I'm not 100% sure how to do this with interfaces, but with regular types, you can do this (I'm oversimplifying):

```
type Props = {
  type: AlertTypes.TOAST,
  delay?: number,
  onClose: () => void
} | {
  type: AlertTypes. RELATIVE | AlertTypes.BANNER,
  onClose?: () => void
}
```